db/migrate/20160613150922_add_event_queue.rb

```diff
@@ -0,0 +1,11 @@
+class AddEventQueue < ActiveRecord::Migration
+  def change
+    create_table :katello_event_queue, :force => true do |t|
```

Comment:
correct.  event gets picked up (marked as in progress), event is executed, event is then deleted form the table.